.github/workflows/macos.yml

```diff
@@ -384,7 +382,7 @@ jobs:
 
       - name: before_deploy
         run: |
-          gem install octokit:'~> 4.15.0' mime-types
+          sudo gem install octokit:'~> 4.15.0' mime-types
```

Comment:
I made this comment in the commit but it has not appeared in the PR so I am adding it again here.

You don't need octokit. It is a dependency of Travis's deployment package "dpl" and at some point in the past we needed a newer version than they had in their standard package. GitHub release deployment is totally different, as far as I am aware.